### PR TITLE
Prepare 0.4.23 release.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Comment by axic** [comment]
As you mentioned the issue with two constructors, should that be considered a bug?

File: docs/bugs.json [hunk 1]
```diff
@@ -1,4 +1,11 @@
 [
+    {
+        "name": "OneOfTwoConstructorsSkipped",
+        "summary": "If a contract has both a new-style constructor (using the constructor keyword) and an old-style constructor (a function with the same name as the contract) at the same time, one of them will be ignored.",
+        "description": "If a contract has both a new-style constructor (using the constructor keyword) and an old-style constructor (a function with the same name as the contract) at the same time, one of them will be ignored. There will be a compiler warning about the old-style constructor, so contracts only using new-style constructors are fine.",
+        "fixed": "0.4.24",
```
Comment (axic) [inline docs/bugs.json]: Should have an introduced line with 0.4.23.

**Review by axic — CHANGES_REQUESTED** [review]

File: docs/bugs.json (hunk 1 above)
Comment (chriseth) [reply docs/bugs.json]: fixed.

File: docs/bugs_by_version.json [hunk 2]
```diff
@@ -502,4 +508,64 @@
         ], 
         "released": "2017-01-31"
     }
+}       "released": "2016-10-31"
+    }, 
+    "0.4.5": {
+        "bugs": [
+            "OneOfTwoConstructorsSkipped",
```
Comment (axic) [inline docs/bugs_by_version.json]: This seems to be broken.

File: docs/bugs.json [hunk 3]
```diff
@@ -1,4 +1,12 @@
 [
+    {
+        "name": "OneOfTwoConstructorsSkipped",
+        "summary": "If a contract has both a new-style constructor (using the constructor keyword) and an old-style constructor (a function with the same name as the contract) at the same time, one of them will be ignored.",
+        "description": "If a contract has both a new-style constructor (using the constructor keyword) and an old-style constructor (a function with the same name as the contract) at the same time, one of them will be ignored. There will be a compiler warning about the old-style constructor, so contracts only using new-style constructors are fine.",
+        "introduced": "0.4.23",
+        "fixed": "0.4.24",
```
Comment (axic) [inline docs/bugs.json]: Oh also these two are broken: 0.4.22 and 0.4.23 :)

File: docs/bugs_by_version.json (hunk 2 above)
Comment (chriseth) [reply docs/bugs_by_version.json]: hm, something is wrong with the script...

**Review by axic — DISMISSED** [review]

**Comment by chriseth** [comment]
The tests fail for me, so let's see....

**Comment by chriseth** [comment]
Ok, it seems we are fine and it is just a local problem.

**Comment by chriseth** [comment]
Argh, it seems it failed for the same reason: `Checking that the bug list is up to date...`

**Comment by chriseth** [comment]
Ok, now it should be working...

**Pull request merged**